src/app/hooks/useAllSkills.ts

```diff
@@ -0,0 +1,15 @@
+import type { Skill } from '../../types';
+import useFetch from './useFetch';
+
+export default function useAllSkills(): {
```

Comment:
Couldn't it just be named `useSkills`? I could also imagine rewriting this hook later to return either a specific or all skills :)